pkg/framework/plugins/podlifetime/pod_lifetime_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/tools/events"
+	utilpointer "k8s.io/utils/pointer"
```

Comment:
This util was deprecated actually and should be replaced by k8s.io/utils/ptr.

Reply:
Done, there are a lot of references to utilpointer still, but will track as a separate PR

Reply:
Thanks, that LGTM to me.